charts/model-engine/templates/istio-virtualservice.yaml

```diff
@@ -12,13 +12,9 @@ metadata:
   {{- end }}
 spec:
   hosts:
-  {{- range .Values.virtualservice.hostDomains }}
-    - "{{ $fullName }}.{{ . }}"
-  {{- end }}
+    - model-engine.{{ $.Values.global.networking.internalDomain }}
   gateways:
-  {{- range .Values.virtualservice.gateways }}
-    - {{ . | quote }}
-  {{- end }}
+    - {{ $.Values.global.networking.internalGateway }}
```

Comment:
im open to scrapping this- this was part of a DNS refactor we did to make specification of virtual services simpler

Reply:
curious to see what the rest of the dns refactor changes look like

Reply:
i made the rest of the refactor- it helps when model engine is a subchart to prevent duplication of changing the core domain a bunch of places